tnoodle-ui/src/main/components/EntryInterface.jsx

```diff
@@ -11,6 +11,7 @@ import { FaEye, FaEyeSlash } from "react-icons/fa";
 const mapStateToProps = (store) => ({
     editingDisabled: store.editingDisabled,
     competitionName: store.wcif.name,
+    generatingScrambles: store.generatingScrambles,
```

Comment:
[Travis](https://travis-ci.org/github/thewca/tnoodle/builds/742319562#L373) is complaining that this is duplicated. Could you please fix it?